client/landing/stepper/declarative-flow/internals/steps-repository/goals/select-goals.tsx

```diff
@@ -18,6 +20,18 @@ export const SelectGoals: React.FC< SelectGoalsProps > = ( {
 	const translate = useTranslate();
 	const goalOptions = useGoals();
 
+	const comingFromCreateBlogLanding = getQueryArgs()?.ref === 'create-blog-lp';
```

Comment:
Just for future-proofing, what do you think about putting this into some referral goals mapping.
```
const someReferralGoals = {
    'create-blog-lp': [ SiteGoal.Write ],
    'something-in-the-future': [ SiteGoal.Sell, SiteGoal.Promote ]
}
```

Reply:
There's only `create-blog-lp` ref available currently, right? 🤔 

Is there any plan to have other refs in place already? Or I guess there's high chance we might need other refs in the future?

Reply:
Yup, there might be one for build and sell, e.g. https://wordpress.com/website-builder/ this one is for build.

But not something to worry for now. Future-proofing helps turn this into a one-line effort in the future.